contracts/interfaces/decent/deployables/IAzoriusV1.sol

```diff
@@ -101,18 +82,20 @@ interface IAzoriusV1 {
     function updateExecutionPeriod(uint32 _executionPeriod) external;
 
     /**
-     * Submits a new Proposal, using one of the enabled [BaseStrategies](../BaseStrategy.md).
+     * Updates the strategy for a Proposal.
+     *
+     * @param _strategy new strategy
+     */
+    function updateStrategy(address _strategy) external;
+
+    /**
+     * Submits a new Proposal.
      * New Proposals begin immediately in the `ACTIVE` state.
      *
-     * @param _strategy address of the BaseStrategy implementation which the Proposal will use
-     * @param _data arbitrary data passed to the BaseStrategy implementation. This may not be used by all strategies,
-     * but is included in case future strategy contracts have a need for it
      * @param _transactions array of transactions to propose
      * @param _metadata additional data such as a title/description to submit with the proposal
      */
     function submitProposal(
-        address _strategy,
-        bytes memory _data,
```

Comment:
Might be prudent to keep this `_data` param, in case a `StrategyV2` in the future needs it. We can of course also have a DAO _update_ their `AzoriusV1` to an `AzoriusV2` if we do decide to need this in the future, but adding it back in now would save that headache.

Reply:
fixed